Seasonal Spotlights: Winter 2018 article (#2137)
ppy/osu-wiki: news/2019-03-31-beatmap-spotlights-winter-2018.md

---
layout: post

> title: "Seasonal Spotlights: Winter 2018"

Reviewer: change winter 2018 to winter 2019

date: 2019-03-31 22:00:00 +0000
---

> Meanwhile the wind blows warmer and the sun feels nice, we still have some beatmaps for you to feel cool. Do not be afraid though, come and take a look at our selection of mapsets from the Winter Season and earn a fancy new medal for yourself!

Reviewer: ```suggestion
The wind's blowing warmer and the sun's feeling nicer, but we still have some cool beatmaps from the winter to show off! Come take a look at our selection of mapsets from last season, and earn a fancy new medal for yourself by playing them all!
```
fixing the grammar alone yields a pretty awkward sentence so I re-worded it a bit too

![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/header.jpg)

> The beatmap charts for the Autumn Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=259). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Winter 2018 Spotlights rankings will end on the 15th of June, 2019.**

Reviewer 2: ```suggestion
The beatmap charts for the Winter Spotlights can be found [here](https://osu.ppy.sh/rankings/osu/charts?spotlight=259). Their respective packs are available at the [Beatmap Pack Listing](https://osu.ppy.sh/beatmaps/packs?type=chart) for easier access. **The Winter 2018 Spotlights rankings will end on the 15th of June, 2019.**
```

Not afraid to accentuate even the tiniest of changes in the song, [cRyo\[iceeicee\]](https://osu.ppy.sh/users/835998) & co. delivers a bombastic take on Reol's song Saisaki. The shift between technical styled mapping for the EDM aspects of the song to the careful arrangements in the melody focused instrumentals and passionately delivered vocals this whole set will not fail to deliver an exhilarating experience. 

Each map is in itself tied together gracefully enriching the song taking it to greater heights, and with a generous spread of difficulty the set is giving every player a change to to enjoy this masterpiece!

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/kyoumen.jpg)](https://osu.ppy.sh/beatmapsets/764481)

[Smokeman](https://osu.ppy.sh/users/2140676) has always been a master of capturing the beauty in the songs he maps. His rendition of "Kyoumen no Nami [Orchestra Ver.] is no different. The map perfectly matches the very serene parts of the song with low density and low slider velocity. Then explodes with a quicker slider velocity and more difficult movements in the chorus which truly make the map shine. 

> Don't forget to look at the wonderous storyboard made by [LowAccuracySS](https://osu.ppy.sh/users/7306763) & [Darky1](https://osu.ppy.sh/users/3372928)!

Reviewer 2: ```suggestion
Don't forget to look at the wondrous storyboard made by [LowAccuracySS](https://osu.ppy.sh/users/7306763) & [Darky1](https://osu.ppy.sh/users/3372928)!
```

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/gin.jpg)](https://osu.ppy.sh/beatmapsets/902948)

[rrtyui](https://osu.ppy.sh/users/352328) brings us a new test of skill on "Gin no Kaze" by wa. remixed celas, sparking a new life into the ideas of Fear's version. As the song slowly reveals itself, the map follows suit to prepare the player for its climax. There, the small circle size is pushed to its limits with an incessant series of unusually precise slider motions. 

Even though hard, this map is also subtle. Combining clever mapping decisions worthy of an essay, rrtyui shows that they have a deep understanding of all layers in mapping. Those who need to improve a bit more to take on the highest difficulty can definitely find a similar challenge in [Uberzolik](https://osu.ppy.sh/users/1314547)'s another.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/first-town.jpg)](https://osu.ppy.sh/beatmapsets/888513)

> Staying true to the chiptune-y video game style of Camellia's First Town Of This Journey, [ScubDomino](https://osu.ppy.sh/users/8972308) plasters densely syncopated slider patterning on a beautifully scenic 8-bit background. Every difficulty, featuring guest mappers [Milan-](https://osu.ppy.sh/users/1052994), [Sing](https://osu.ppy.sh/users/3795679), and [Delis](https://osu.ppy.sh/users/1603923), follows similar conventions, using sliderstreams and overlaid sliders to give the entire mapset a cohesive motif culminating in a spectacular interpretation through ScubDomino's [Adventure] difficulty. 

Author: ```suggestion
Staying true to the chiptune-y video game style of Camellia's First Town Of This Journey, [ScubDomino](https://osu.ppy.sh/users/8972308) plasters densely syncopated slider patterning on a beautifully scenic 8-bit background he drew himself. Every difficulty, featuring guest mappers [Milan-](https://osu.ppy.sh/users/1052994), [Sing](https://osu.ppy.sh/users/3795679), and [Delis](https://osu.ppy.sh/users/1603923), follows similar conventions, using sliderstreams and overlaid sliders to give the entire mapset a cohesive motif culminating in a spectacular interpretation through ScubDomino's [Adventure] difficulty. 
```

> The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these contraints in a magnificent climax rich with sliderstreams and velocity changes that slowly reverts back to its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.

Reviewer 2: ```suggestion
The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these constraints in a magnificent climax rich with sliderstreams and velocity changes that slowly reverts back to its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.
```

> The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these constraints in a magnificent climax rich with sliderstreams and velocity changes that slowly reverts back to its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.

Reviewer 3: "ScubDomino plasters densely syncopated slider patterning **on a beautifully scenic 8-bit background he drew himself.**"

"in a magnificent climax rich with sliderstreams and velocity changes **and finally reverts back to** its more structured composition"

Just wanting to make a couple edits to the description I wrote.

Reviewer 2: Use this:

![](https://puu.sh/D7gd5/3c18af5746.png)

Reviewer 3: Should I use that too if the mappers in the graphic for First Town of this Journey aren't capitalized?

Reviewer 2: You can only change text with that, so a regular comment is fine.

> The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these constraints in a magnificent climax rich with sliderstreams and velocity changes that slowly reverts back to its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.

Author: ```suggestion
The map itself follows its own journey, beginning with highly grid-oriented patterning that slowly diverges from these constraints in a magnificent climax rich with sliderstreams and velocity changes and finally reverts back its more structured composition as the music also returns home. Find yourself grooving along to this bright, upbeat tune as you embark on your personal journey through this mapset.
```

Another highlight of the set is definitely the high effort hitsounding which adds a great hit-feedback for an even better game-play experience. If you can not play the top difficulty, the insane gives out an equal smooth atmosphere so its worth on checking them both out.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/icarus.jpg)](https://osu.ppy.sh/beatmapsets/766441)

[KKipalt](https://osu.ppy.sh/users/6889573) perfectly captures the intensity of EDEN's Icarus with exciting bursts and captivating rhythms which makes playing the slow parts completely worth it. As a person with such a small attention span it's not common that I stick around for an entire 6 min map let alone a slow one.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/osu/hyperdrive.jpg)](https://osu.ppy.sh/beatmapsets/883505)

> It's the first time [hypercyte](https://osu.ppy.sh/users/9155377) is in the Spotlights, and he's here with a rockin' mapset: the storyboard is really well done, fitting nicely with the map and song, and the patterns are somehow original, making it enjoyable to play and standing out among all these usual pp/tech maps.

Author: ```suggestion
HYPERDRIVE, [hypercyte](https://osu.ppy.sh/users/9155377)'s first spotlighted map, marks a departure from his usual anime TV size sets. Featuring a dynamic and creative storyboard set to an energetic electronic melody,  HYPERDRIVE takes the player on a voyage into the galaxy and beyond. Dive into the HYPERNOVA difficulty and experience engaging jumps, intricate slider patterns and well-thought-out designs sure to express the song to its fullest. For all you aspiring spaceship pilots out there, be sure to take a look at the lower difficulties made by [Peter](https://osu.ppy.sh/users/8623835) and [Dashyy-](https://osu.ppy.sh/users/6513356), as well as hypercyte's own Easy difficulty.
```

It does not end here, though. The beatmap itself is challenging on its own! Apart from the concept mapping, Kazu also made sure to squeeze every element of the music as much as possible, resulting in several mixed snaps (it's still breakcore) that will keep all players on their toes!

All in all, for the sheer capacity of interpretation, the outstanding musical ear the mapper shows with this creation, and the beatmap itself being also incredibly fun to play, Feline, The White... earns its spot on these seasonal Spotlights very fairly!

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/give-it-up.jpg)](https://osu.ppy.sh/beatmapsets/618352)

> Britland makes the spotlight this season, as [Grimbow](https://osu.ppy.sh/users/1387762) proudly repesents them with his map of a remix of Knife Party's "Give It Up." While the song shoud poise some headbanging from each and every player, the map offers an experience that expert-level players will come to enjoy no matter how much they play it!

Reviewer 2: ```suggestion
Britland makes the spotlight this season, as [Grimbow](https://osu.ppy.sh/users/1387762) proudly represents them with his map of a remix of Knife Party's "Give It Up." While the song should poise some headbanging from each and every player, the map offers an experience that expert-level players will come to enjoy no matter how much they play it!
```

This map is a great test to your stamina and consistency, and likely a fair challenge to everyone capable of playing this on Double Time! All in all, Volta made a really enjoyable honest tea that deserves its spot in here very well!

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/impulse.jpg)](https://osu.ppy.sh/beatmapsets/751401)

There are some mappers that don't aim a lot of their content to the ranked section, but when they do, they never cease to amaze us with fresh innovative content. Of course one of them is [Fallmorph](https://osu.ppy.sh/users/5720389)! The mapper who amazed us with Enantiomorphs three seasons ago, has come back with a new mind-blowing Marathon map, making an amazing mapping rendition to Impulse, a futuristic sounding drumstep track by osu!'s Featured Artist, Culprate.

This map is absolutely worth of being called official content, since it includes high effort mapping skills such as amazing control of contrast between sections, climatic tension, advanced slider velocity changes and hitsounding, the latter being absent of most osu!taiko beatmaps.

> Impulse is a dreamy journey through a futuristic utopia, with calm, high spaced notes falling into interesting drum beats, playing a lot with rhythm variance and clever usage of drumrolls, beatifully executed along with synth hitsounded big note ornaments.

Reviewer 2: ```suggestion
Impulse is a dreamy journey through a futuristic utopia, with calm, high spaced notes falling into interesting drum beats, playing a lot with rhythm variance and clever usage of drumrolls, beautifully executed along with synth hitsounded big note ornaments.
```

1/3? 1/4? 1/6? 1/8? This map features them all! As we go deeper into this atmosphere of textures, we find ourselves falling progressively into a more difficult, technical style of mapping: The track goes on, but it doesn't get any boring, it just gets better and better...

> Saying any more about this map could be considered an spoiler, so how about you give it a try? Trust us, it won't dissapoint you!

Reviewer 2: ```suggestion
Saying any more about this map could be considered an spoiler, so how about you give it a try? Trust us, it won't disappoint you!
```

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/taiko/epimerization.jpg)](https://osu.ppy.sh/beatmapsets/754062)

This beatmap by [Alchyr](https://osu.ppy.sh/users/4993032) is completed as a single story. It begins with the first silent scene following the quiet melody. after that, the story changes dramatically, just like a powerful storm of finishers and spinners. The visual difficulty of finishers further enhances it. Then the storm turns into something like 1/6 arrows, and we can easily experience how difficult handswaping is. 

After a long break, you can see a very impressive part making use of tremendous density changes. In the end, the story finally finishes with the perfect part summarizing all its concepts. The great attractive thing about this story is that it is not boring but inversely enjoyable despite repeating the same pattern, of course because of the perfection of mapping.

> まさしく，一つの物語として成す譜面です．まず，物語は静寂たる音色とそれに合った静けさに始まります．束の間，まるで大音符とスピナーの嵐かのように劇的に物語は変化します．大音符の視認性は拍車をかかけます．そして嵐は1/6の矢へと移り変わり，逆手の極みを篤と見せつけてくるのです．長い静寂を挟み，強烈な密度変化を取り入れた，凄まじく印象的な場面が広がると，最後はこれまでのすべてのコンセプトを合わせた場面によって物語は終焉を迎えます．同じ配置を繰り返しているようで，つまらぬどころか面白い，それはもちろん，譜面制作の極致に至るこの譜面の持つ趣のまさに一つと言えましょう．

Reviewer 4: can some context be given as to why there's a Japanese paragraph

Author: Toorun asked me to include it as well. It is the Japanese version of the comment. Personally, I see no harm in including it.

From the victors of UCMC 2, [JBHyperion](https://osu.ppy.sh/users/4879508) and [Sorcerer](https://osu.ppy.sh/users/1913190)’s collab rendition of lapix’s "Good Time" will give you a damn good time while playing as the two mappers’ styles blend cohesively together. Featuring a low bpm, high rhythm density, and tough dashing, this set will keep you on your toes for the majority of the song’s five minutes.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/catch/eden.jpg)](https://osu.ppy.sh/beatmapsets/810279)

Bringing a more unconventional style to the table, [alienflybot](https://osu.ppy.sh/users/636114)’s mapset EDEN used uncommon patterns such as frequent edge dashes, high ratio of circles to sliders, and low approach rate to create a unique challenge for players looking for something different and memorable.

In addition, [Ascendance](https://osu.ppy.sh/users/2931883)’s difficulty is a more conventional approach to mapping a song like this and makes a solid addition to complete the set.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/catch/brionac.jpg)](https://osu.ppy.sh/beatmapsets/734296)

> As one of the top players in osu!catch, [Rocma](https://osu.ppy.sh/users/566276) has been trying to map technical maps for top players for long time. Although Brionic is his first ranked map, the quality of the map proves that he is also a good mapper with the ability to create a whole set. The top diff may have a lower star rating comparing to other maps, but the anti-flow 1/4 streams are still pretty hard to deal with even if you are a top player. 

Reviewer 2: ```suggestion
As one of the top players in osu!catch, [Rocma](https://osu.ppy.sh/users/566276) has been trying to map technical maps for top players for long time. Although Brionac is his first ranked map, the quality of the map proves that he is also a good mapper with the ability to create a whole set. The top diff may have a lower star rating comparing to other maps, but the anti-flow 1/4 streams are still pretty hard to deal with even if you are a top player. 
```

The balance in this map between serenity and chaos perfectly represents the music it accompanies. For this reason, the mapset has a clearly well-deserved spot in the spotlights, and we will definitely expect more from Dergo in the future.

[![](/wiki/shared/news/2019-03-31-beatmap-spotlights-winter-2018/mania/relentless.jpg)](https://osu.ppy.sh/beatmapsets/881771)

Are you ready for the ruffneck bass? As a skillful osu!mania mapper and Beatman Nominator who brought the ranked section with a few delicate 4K spreads in the past, [_Stan](https://osu.ppy.sh/users/1653229) this time returns with an unexpected 7K marathon of this exciting piece of electronic music, DJ Noriken - #The_Relentless(Modified), that testifies to his deep understanding of 7K mapping as well.

> This marathon is mostly composed of normal notes, but in a way that fully convey every single line of music, making it actually more interesting and beginner-friendly. However, it is challenging at the same time, since _Stan creates it in a way that gradually poses more difficulty onto the map. At the last kiai, the full mixture of short LNs with streams greatly enhances the overall difficulty of the map to mark the pinnacle of his understanding over the progression of the music, while it also necessarily avoids SR inflation.

Reviewer: ```suggestion
This marathon is mostly composed of normal notes, but in a way that fully convey every single line of music, making it actually more interesting and beginner-friendly. However, it is challenging at the same time, since \_Stan creates it in a way that gradually poses more difficulty onto the map. At the last kiai, the full mixture of short LNs with streams greatly enhances the overall difficulty of the map to mark the pinnacle of his understanding over the progression of the music, while it also necessarily avoids SR inflation.
```

> In this way, this mapset deserves a place in the spotlight and we sincerely look forward to _Stan's future 7K creations!

Reviewer: ```suggestion
In this way, this mapset deserves a place in the spotlight and we sincerely look forward to \_Stan's future 7K creations!
```
